application/comit_node/src/node_id.rs

```diff
@@ -0,0 +1,2 @@
+use std::net::SocketAddr;
+pub type NodeId = SocketAddr;
```

Comment:
:question: Should it really be a type alias and not a new type? Given that we might change this at some point, a new type would be more resilient to refactoring?

Reply:
I just looked at doing this and actually it wouldn't save any work later down the track because it will completely change with the introduction of noise IDs.